apps/tasks/lib/components/task_card.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:flutter/material.dart';
+import 'package:helpwave_localization/localization.dart';
+import 'package:helpwave_theme/constants.dart';
+import 'package:helpwave_widget/static_progress_indicator.dart';
+import 'package:tasks/dataclasses/task.dart';
+
+class TaskCard extends StatelessWidget {
+  final TaskWithPatient task;
+  final EdgeInsetsGeometry? margin;
+  final double borderRadius;
+
+  const TaskCard({super.key, required this.task, this.margin, this.borderRadius = borderRadiusMedium});
+
+  String getDueText(BuildContext context) {
+    String result = context.localization!.dueIn;
```

Comment:
Please remove the unnecessary else

Reply:
and i think it would make sense use a switch statement instead

Reply:
`class MyLocalizations {
  static String minutes(int count) => Intl.plural(
    count,
    one: '1 minute',
    other: '$count minutes',
    name: 'minutes',
  );
}`